src/sass/components/Header.scss

```diff
@@ -0,0 +1,12 @@
+.Header {
```

Comment:
the margin and padding aren't actually required for the fix, but they help collapse the wrapper to the content, which I think it appropriate here.

The other change is just setting a defined width so the `auto` width doesn't recalculate during load.